porcupine/plugins/find.py

```diff
@@ -145,12 +145,7 @@ def show(self) -> None:
         try:
             selected_text: Optional[str] = self._textwidget.get("sel.first", "sel.last")
         except tkinter.TclError:
-            selected_text = None
-
-        self.pack(fill="x")
-
-        if selected_text is None or "\n" in selected_text:
-            self.find_entry.focus_set()
```

Comment:
I would rather not delete all this :) Can you add it back?

Reply:
WTF?

Reply:
.... git diff --staged .... I never learn .